packages/paste-core/layout/grid/src/utils.tsx

```diff
@@ -1,16 +1,86 @@
-import {ResponsiveValue} from '@twilio-paste/styling-library';
-import {Margin, Space, SpaceOptions} from '@twilio-paste/style-props';
-import {
+import type {ResponsiveValue} from '@twilio-paste/styling-library';
+import type {Margin, Padding, Space, SpaceOptions} from '@twilio-paste/style-props';
+import type {
   ColumnOffset,
   ColumnOffsetOptions,
   ColumnProps,
   ColumnSpanOptions,
   ColumnMinWidth,
   ColumnWidthSpan,
+  ColumnPadding,
 } from './types';
 
 type Vertical = ResponsiveValue<boolean>;
 
+const getSpaceDefault = (responsive = false): 'space0'[] | 'space0' =>
+  responsive ? new Array(3).fill('space0') : 'space0';
+
+const initDefaultSpacing = ({
+  prefix,
+  responsive,
+}: {
+  prefix: string;
+  responsive: boolean;
+}): {[x: string]: Margin | Padding} => ({
```

Comment:
Might just be me, but I'm finding these inline typings on these method are making things pretty hectic to read. Is it worth pulling them out into types and interfaces to clean the syntax up a little?

Reply:
Sure thing, I can make that happen. I think this shape is reused quite a bit too, so it would be worth creating a named type anyway. Thanks!